docs/cql/dml.rst

```diff
@@ -20,7 +20,7 @@ sections common to data updating statements.
 Update parameters
 ~~~~~~~~~~~~~~~~~
 
-The ``UPDATE``, ``INSERT`` (and ``DELETE`` and ``BATCH`` for the ``TIMESTAMP``) statements support the following
+The ``UPDATE``, ``INSERT`` (and ``DELETE`` and ``BATCH`` for the ``TIMESTAMP`` and ``TIMEOUT``) statements support the following
 parameters:
```

Comment:
We might as well rephrase this entire sentence. It is min-bendingly difficult to follow.

Reply:
Agree but we don't need to do it now.

Reply:
ok, I'm fine with a follow-up.